From 6616bf5b14d3479946a8557ab4226ab4f55fbb47 Mon Sep 17 00:00:00 2001
From: Sebastian Pipping <sebastian@pipping.org>
Date: Thu, 2 Jul 2020 00:18:54 +0200
Subject: [PATCH 09/11] AppVeyor: Get list of tox environments back in sync
 with tox.ini

In detail:
- drop tox environments that do not currently exist in tox.ini
- cover Django 2.0 and 2.1 as well
---
 appveyor.yml | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/appveyor.yml b/appveyor.yml
index 84b5734..0514065 100644
--- a/appveyor.yml
+++ b/appveyor.yml
@@ -3,12 +3,9 @@ services:
 
 environment:
   matrix:
-    - TOXENV: py27-dj18-mssql
-    - TOXENV: py27-dj19-mssql
-    - TOXENV: py27-dj110-mssql
-    - TOXENV: py35-dj19-mssql
-    - TOXENV: py36-dj110-mssql
     - TOXENV: py36-dj111-mssql
+    - TOXENV: py36-dj20-mssql
+    - TOXENV: py36-dj21-mssql
 
 matrix:
   fast_finish: true
-- 
2.25.4

